.github/workflows/backport.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: Backport Assistant Runner
+
+on:
+  pull_request:
+    types:
+      - closed
+      - labeled
+
+jobs:
+  backport:
+    if: github.event.pull_request.merged
+    runs-on: ubuntu-latest
+    container: hashicorpdev/backport-assistant:0.2.5
+    steps:
+      - name: Run Backport Assistant
+        run: backport-assistant backport
```

Comment:
Do we want to have to intervene with every backport PR or would we like it to automerge if there are no issues?

This option is available: `backport-assistant backport -automerge`

Reply:
Talked to Ashwin about this, will make this change